src/test/scala/labs/Lab1Test.scala

```diff
@@ -96,12 +96,33 @@ class SingleCycleAddTesterLab1 extends CPUFlatSpec {
 class SingleCycleRTypeTesterLab1 extends CPUFlatSpec {
   behavior of "Single Cycle CPU"
   for (test <- InstTests.rtype) {
-    it should s"run R-type instruction ${test.binary}${test.extraName}" in {
-      CPUTesterDriver(test, "single-cycle") should be(true)
+    if (test.binary !== "add0") {
```

Comment:
I'd say this should either be `if(test.binary !== "add0" || test.binary !== "add1")` or we could just run the tests multiple times. The lab should say "run add1" then "run add0" then "run all R type". So, if the students get here add1 and add0 should implicitly work.

Reply:
In the graded add test, add2 is included but not in the labs version. Should they be brought to parity? Or is it fine to just ignore add2 in the graded test version like add0 and add1, but keep it in the labs version?

Reply:
Bringing them up to parity would be best. More tests is better :).